docs/fsharp/whats-new/fsharp-50.md

```diff
@@ -401,7 +401,7 @@ This feature implements [F# RFC FS-1074](https://github.com/fsharp/fslang-design
 
 ## Simplified interop with nullable value types
 
-[Nullable (value) types](https://docs.microsoft.com/dotnet/api/system.nullable-1) (called Nullable Types historically) have long been supported by F#, but interacting with them has traditionally been somewhat of a pain since you'd have to construct a `Nullable` or `Nullable<SomeType>` wrapper every time you wanted to pass a value. Now the compiler will implicitly convert a value type into a `Nullable<ThatValueType>` if the target type matches. The following code is now possible:
+[Nullable (value) types](/dotnet/api/system.nullable-1) (called Nullable Types historically) have long been supported by F#, but interacting with them has traditionally been somewhat of a pain since you'd have to construct a `Nullable` or `Nullable<SomeType>` wrapper every time you wanted to pass a value. Now the compiler will implicitly convert a value type into a `Nullable<ThatValueType>` if the target type matches. The following code is now possible:
```

Comment:
```suggestion
[Nullable (value) types](xref:System.Nullable`1) (called Nullable Types historically) have long been supported by F#, but interacting with them has traditionally been somewhat of a pain since you'd have to construct a `Nullable` or `Nullable<SomeType>` wrapper every time you wanted to pass a value. Now the compiler will implicitly convert a value type into a `Nullable<ThatValueType>` if the target type matches. The following code is now possible:
```

Reply:
@gewarren I had to revert this as Markdownlint didn't like your change.

Reply:
What was the error? That should be a valid link, and xref is preferred for API links.

`[Nullable (value) types](xref:System.Nullable`1)` is what's generated by the Docs markdown extension.

Reply:
No idea why.  My guess is an escaping issue in markdownlint with the ``` ` ```.  OPS thought it was fine, it was markdownlint that didn't like it and gave a funny "No HTML elements" message.